prov/bgq/configure.m4

```diff
@@ -0,0 +1,91 @@
+dnl Configury specific to the libfabrics BGQ provider
+
+dnl Called to configure this provider
+dnl
+dnl Arguments:
+dnl
+dnl $1: action if configured successfully
+dnl $2: action if not configured successfully
+dnl
+AC_DEFUN([FI_BGQ_CONFIGURE],[
+	# Determine if we can support the bgq provider
+	bgq_happy=0
+	bgq_direct=0
+
+	bgq_driver=/bgsys/drivers/ppcfloor
+	AC_SUBST(bgq_driver)
+	AC_ARG_WITH([bgq-driver],
+		[AS_HELP_STRING([--with-bgq-driver=@<:@BGQ driver installation path@:>@],
+			[Provide path to where BGQ system headers are installed])
+		],
+		[bgq_driver=$with_bgq_driver])
+
+	bgq_driver_CPPFLAGS="-I$bgq_driver -I$bgq_driver/spi/include/kernel/cnk"
+	CPPFLAGS="$bgq_driver_CPPFLAGS $CPPFLAGS"
+
+	AS_IF([test x"$enable_bgq" != x"no"],
+		[AC_CHECK_HEADER(hwi/include/bqc/MU_Descriptor.h,
+			[bgq_happy=1],
+			[bgq_happy=0])])
+
+	bgq_external_source=auto
+	AC_SUBST(bgq_external_source)
+	AC_ARG_WITH([bgq-src],
+		[AS_HELP_STRING([--with-bgq-src(=DIR)],
+			[bgq opensource distribution @<:@default=auto@:>@])
```

Comment:
This is a total nit-pick, sorry: the capitolization of this help string isn't consistent with the help string that you provided above.